String/test/LengthofLongestSubstringWithoutRepetition.test.js

```diff
@@ -0,0 +1,32 @@
+import { lengthOfLongestSubstring } from '../LengthofLongestSubstringWithoutRepetition'
+
+describe('Testing the lengthOfLongestSubstring function', () => {
```

Comment:
These tests are fine, but please remove some redundant wording. Everything but `lengthOfLongestSubstring` is unnecessary here. The `Test N: Check` below is unnecessary as well.

Reply:
Done